<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Renovacao</class>
 <widget class="QDialog" name="Renovacao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RENOVAÇÃO</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NOME DO LEITOR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NOME">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NOME DO LIVRO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NOME_LIVRO">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NOME DO AUTOR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NOME_AUTOR">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PESQUISAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DIAS PARA ENTREGA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>DATA DA RENOVAÇÃO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCalendarWidget" name="calendarWidget"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RENOVAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ÍNDICE DO LEITOR </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="INDICE"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
